src/Details.js

```diff
@@ -1,11 +1,29 @@
-import React from 'react';
+import React, {Component} from 'react';
 import { Link } from 'react-router-dom';
 
-export default function Details() {
-    return (
-        <div>
-            <p>Hello, this will be the details page for each Movie & TV show :)</p>
-            <Link to='/'>Back to home</Link>
-        </div>
-    );
+export default class Details extends Component {
+    constructor() {
+        super();
+        this.state = {
+            welcomeMessage: 'Welcome to the details, WIP :)'

```

Comment:
you can remove the space between `this.state = {` and the welcomeMessage

Reply:
Seems to be a issue with github showing the code, here it seems fine but when 'view changes' clicked theres a space between these two lines of code. No need for changes here :)